docs/sql-ref-syntax-ddl-drop-view.md

```diff
@@ -19,4 +19,64 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+Drops the specified `VIEW`, which was created by `CREATE VIEW` statement. `DROP VIEW` only involves
```

Comment:
How about the following ?
DROP VIEW` removes the metadata associated with a specified view from the catalog.